PragmaThemePlugin.inc.php

```diff
@@ -87,11 +89,37 @@ public function init()
         $this->addStyle('htmlGalley', 'resources/less/import.less', ['contexts' => 'htmlGalley']);
         $this->modifyStyle('htmlGalley', ['addLessVariables' => join("\n", $additionalLessVariables)]);
 
+        HookRegistry::add('TemplateManager::display', [$this, 'initializeTemplate']);
         HookRegistry::add('TemplateManager::display', [$this, 'addSiteWideData']);
         HookRegistry::add('TemplateManager::display', [$this, 'addIndexJournalData']);
         HookRegistry::add('TemplateManager::display', [$this, 'checkCurrentPage']);
     }
 
+    /**
+     * Initialize Template
+     */
+    public function initializeTemplate(string $hookname, array $args): bool
+    {
+        $templateMgr = $args[0];
+        // The login link displays the login form in a modal, therefore the reCAPTCHA/ALTCHA must be available for all frontend routes
+        $isCaptchaEnabled = Config::getVar('captcha', 'recaptcha') && Config::getVar('captcha', 'captcha_on_login');
+        if ($isCaptchaEnabled) {
+            $templateMgr->addJavaScript(
+                'recaptcha',
+                'https://www.recaptcha.net/recaptcha/api.js?hl=' . substr(Locale::getLocale(), 0, 2)
```

Comment:
Should `substr` still be used here? I see in `PKPTemplateManager` we no longer use it as of 3.5: https://github.com/pkp/pkp-lib/blob/ff423896b6459b98b4ab96444175c751f7dd821c/classes/template/PKPTemplateManager.php#L264